docs/content/test_runner/Metrics.md

```diff
@@ -28,4 +28,9 @@ Time from first test execution start to last test execution finished
 
 Total job time (suite time with both delays)
 
+### `device-utilization.median`
+
+Effective device work time relative to total device claimed time, in percent `0-100` \
```

Comment:
> Effective device work time

It's not clear what is "effective time". It'd be great to visualize it.

Reply:
How do you like it?

![image](https://user-images.githubusercontent.com/1105133/106201847-1fd0d580-61ca-11eb-93f2-32306a56734c.png)

Reply:
Eccellente!